src/main.rs

```diff
@@ -171,7 +171,7 @@ fn run_spec(spec_text: &str, args: &[String], opts: Options) {
             print!(" ");
         }
 
-        let our_html = render_html(&test.input.replace("→", "\t"), opts);
+        let our_html = render_html(&test.input.replace("→", "\t").replace("\n", "\r\n"), opts);
```

Comment:
The normalization I'm referring to happens in prototype.rs in the parse_indented_code_block function when I strip off line endings and add a SynthesizeNewline node to the tree.